pkg/apis/elasticsearch/v1/elasticsearch_config.go

```diff
@@ -8,17 +8,17 @@ import (
 	"github.com/elastic/go-ucfg"
 	"k8s.io/utils/pointer"
 
+	commonv1 "github.com/elastic/cloud-on-k8s/pkg/apis/common/v1"
 	"github.com/elastic/cloud-on-k8s/pkg/controller/common/version"
 	"github.com/elastic/cloud-on-k8s/pkg/utils/stringsutil"
-
-	commonv1 "github.com/elastic/cloud-on-k8s/pkg/apis/common/v1"
 )
 
 type NodeRole string
 
 const (
 	DataColdRole            NodeRole = "data_cold"
 	DataContentRole         NodeRole = "data_content"
+	DataFrozenRole          NodeRole = "data_frozen"
```

Comment:
Not sure how we missed the frozen role and did not notice for so long. Should we also add the matching label to the pods for consistency?

Reply:
`elasticsearch.k8s.elastic.co/node-data_frozen` added in https://github.com/elastic/cloud-on-k8s/pull/5248/commits/ac81b733e712f96fc0dd6d601a4f4350ba784e2b